src/Import/ManagerInterface.php

```diff
@@ -20,6 +20,22 @@ interface ManagerInterface {
    */
   public function syncList($sync_type_id);
 
+  /**
+   * Syncs an entity, given an ID, that needs to be updated from the remote.
+   *
+   * Fetches an entity for the given Drupal ID and entity type from the remote
+   * service and syncs it with Drupal.
+   *
+   * @param string $sync_type_id
+   *   The ID of the entity sync type.
+   * @param \Drupal\Core\Entity\EntityInterface $drupal_entity
+   *   The Drupal entity that we are trying to sync.
+   *
+   * @return \Drupal\Core\Entity\EntityInterface
+   *   The Drupal entity that was synced.
+   */
+  public function syncGet($sync_type_id, EntityInterface $drupal_entity);
```

Comment:
**CHANGE REQUEST**

As discussed, this should be `public function importLocalEntity`.

Reply:
Yeah, I didn't get to working on this yesterday.